lib/reversetunnel/agentpool.go

```diff
@@ -95,6 +95,9 @@ type AgentPoolConfig struct {
 	Cluster string
 	// FIPS indicates if Teleport was started in FIPS mode.
 	FIPS bool
+	// LocalAuthServers is a list of auth servers to use when dialing back to
+	// the local cluster.
+	LocalAuthServers []string
```

Comment:
I prefer `LocalAuthAddresses` instead of `LocalAuthServers`, but if we settle on `LocalAuthServers` then we should use that in `AgentConfig` as well.